src/main/resources/anniv5Resources/localization/eng/monsterhunterpack/Cardstrings.json

```diff
@@ -1,7 +1,7 @@
 {
   "${ModID}:HuntingStrike": {
     "NAME": "Hunting Strike",
-    "DESCRIPTION": "Deal !D! damage. NL Deals double damage to Elites and Bosses. NL Draw 1 card."
+    "DESCRIPTION": "Deal !D! damage. NL Draw a card. NL Deals double damage to Elites and Bosses. NL Draw 1 card."
```

Comment:
Uhh, that'd be a repat?

The text alreadys says Draw 1 card at the end

Reply:
wait it does? Oh I am dumba